static/app/views/replays/replays.tsx

```diff
@@ -34,12 +34,19 @@ type Props = AsyncView['props'] &
 class Replays extends React.Component<Props> {
   getEventView() {
     const {location, selection} = this.props;
+    const {query} = location;
+
+    let sort = '-timestamp';
+    if (query.sort) {
+      sort = query.asc ? query.sort : `-${query.sort}`;
```

Comment:
does `query.sort` include the `-` prefix? If it does would it be possible to get into weird situations where we see something like `?sort=---timestamps`?

Reply:
Hi @ryan953 don't think that situation can happen, because I'm only using the `-` prefix as a default for the first time when the user opens the page.
In case we had another sort (in the future) or just sorting by timestamp will override that prefix.